config/initializers/cluster.rb

```diff
@@ -1,11 +1,15 @@
-cluster_yml = YAML.load(File.read('config/cluster.yml'))
-MAINNET_CLUSTER_VERSION = cluster_yml['software_patch_mainnet']
```

Comment:
If you remove a constant, you need to update all places where this constant is being used.

Reply:
@kbiala  Where this constant is being used yet?

Reply:
Run global search in the project. Both MAINNET_CLUSTER_VERSION and TESTNET_CLUSTER_VERSION are used in helpers.

Reply:
After pulling "pythnet-network" branch new occurrences of MAINNET_CLUSTER_VERSION and TESTNET_CLUSTER_VERSION were added.